.github/workflows/build.yml

```diff
@@ -37,6 +37,10 @@ jobs:
         pip install -r dev-requirements.txt
         pip install .
 
+    - name: Install ert-storage
+      run: |
+        pip install git+https://github.com/equinor/ert-storage.git
```

Comment:
Please add a comment as to why it is installed like this and create the necessary issues in both ert-storage and ert to outline getting ert-storage on PyPI and added to ERT's dependencies...

Reply:
done!